### PR TITLE
docs(platform/Gitea): Update PAT scopes for Gitea

### PR DESCRIPTION
**Pull request by CodeShakingSheep**
## Changes

I added the newly required PAT scopes for Gitea which are mandatory since Gitea version 1.20.0 to the documentation. 

## Context

Following this issue https://github.com/renovatebot/renovate/discussions/23606#discussioncomment-6583978 I created this PR for updating required PAT scopes for running renovate for Gitea (>= version 1.20.0) repositories.

## Documentation (please check one with an [x])

- [x] I have updated the documentation, or
- [ ] No documentation update is required

## How I've tested my work (please select one)

I have verified these changes via:

- [x] Code inspection only, or
- [ ] Newly added/modified unit tests, or
- [ ] No unit tests but ran on a real repository, or
- [ ] Both unit tests + ran on a real repository

### CODE REVIEW

File: lib/modules/platform/gitea/index.md [hunk 1]
```diff
@@ -18,7 +18,10 @@ You must set `platform=gitea` in your Renovate config file.
 The PAT should have these permissions:
 
 - `repo`
-- `read:user` and `read:email`
+- `read:user`
+- `read:misc` (since Gitea version 1.20.0)
```
Comment (viceice) [inline lib/modules/platform/gitea/index.md:22]: no longer required since v1.20.2

Comment (CodeShakingSheep) [reply lib/modules/platform/gitea/index.md]: Ok, I updated it in https://github.com/renovatebot/renovate/pull/23615/commits/17e79c0190d361b19e7edd51eb9303573892af29 .

File: lib/modules/platform/gitea/index.md [hunk 2]
```diff
@@ -18,7 +18,10 @@ You must set `platform=gitea` in your Renovate config file.
 The PAT should have these permissions:
 
 - `repo`
-- `read:user` and `read:email`
+- `read:user`
```
Comment (HonkingGoose) [inline lib/modules/platform/gitea/index.md:21]: Can we try a table layout instead? I'm thinking of something like this:

| Permission   | Valid for Gitea versions       |
| ------------ | ------------------------------ |
| `repo`       | all                            |
| `read:user`  | all                            |
| `read:email` | `< 1.19.3`                     |
| `issue`      | `> 1.20.0`                     |
| `read:misc`  | Only for `1.20.0` and `1.20.1` |

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

File: lib/modules/platform/gitea/index.md (hunk 2 above)
Comment (CodeShakingSheep) [reply lib/modules/platform/gitea/index.md]: Love the idea of a table layout. Looks cleaner like this. Done in https://github.com/renovatebot/renovate/pull/23615/commits/36c3af5e4da57cbc5be1c835b1e94e6f85f97cca . I just slightly changed the order to have the PAT scopes for newer versions above the ones for older versions.

File: lib/modules/platform/gitea/index.md [hunk 3]
```diff
@@ -17,8 +17,13 @@ You must set `platform=gitea` in your Renovate config file.
...
 
-- `repo`
-- `read:user` and `read:email`
+| Permission  | Valid for Gitea versions |
+| ----------- | ----------- |
+| `repo`      | all       |
+| `read:user` | all        |
+| `issue`     | `>= 1.20.0`|
+| `read:email`| `<= 1.19.3`|
+| `read:misc` | Only for `1.20.0` and `1.20.1`|
```
Comment (HonkingGoose) [inline lib/modules/platform/gitea/index.md:26]:
```suggestion
| Permission   | Valid for Gitea versions       |
| ------------ | ------------------------------ |
| `repo`       | all                            |
| `read:user`  | all                            |
| `issue`      | `>= 1.20.0`                    |
| `read:email` | `<= 1.19.3`                    |
| `read:misc`  | Only for `1.20.0` and `1.20.1` |
```

Apply this suggestion to fix the Prettier lint test.

If you have a development environment setup, or use GitHub Codespaces, you can also run `yarn prettier-fix` and commit the fixed layout, and push it to your branch.

**Review by HonkingGoose — CHANGES_REQUESTED** [review]

**Review by viceice — APPROVED** [review]

**Pull request merged**